Client-Side/shecodesprogramming/src/Components/HomePage/homepage.jsx

```diff
@@ -0,0 +1,109 @@
+
+function Homepage(){
+    return(
+        <div>
+            <div className="MainLayout">
+                <div className="layout">
+                    <div className="sublayout">
+                        <img className="welcome" src="https://kalvium.community/images/home_hero.gif" alt="" />
+                        <h2></h2>
+                        <p>Welcome to this amazing place where you can have lots of fun and learn new things! It's a special spot where you'll experience the joy of learning and discover the wonders of magic. Enjoy every moment of your journey here!</p>
+                        <p>To begin learning the courses, simply go to Livebooks. It's a platform where you can access educational materials and resources. It's like a virtual library for learning. Once you're there, you'll find a variety of courses and materials to explore and study. It's a great place to start your learning journey and access the knowledge you need. Enjoy your learning experience on Livebooks!</p>
+                    </div>
+
+                    <div className="display">
+                        <div>
+                            <img className="whyimage" src="https://uploads-ssl.webflow.com/620ce06fa12cad2d27fb30a8/630cc6df6db69f0f8c9f313a_1%20flipped-p-800.png" alt="" />
+                        </div>
+                        <div className="text-image">
+                            <h2>Why SHecodes</h2>
+                            <p>In India, 8 lakh students graduate from engineering in every year around 40 lakhs students join some sort of computer science program for a carrier in a computer science.</p>
+                            <p>The sad reality is only 6% of them get placed in software companies.</p>
+                            <p>The fund problem is lack of programming skills.If programming skills can be taught from a young age we can bridge this gap throughthere are many websites to provide knowledge. </p>
+                            <p>There are few websites which bundles knowledge with skill development and most of websites in english.</p>
+                            <p>SHecodes is a web targeted towards telugu speaking students. SHecodes aims to empower every telugu medium students to become a solid programmer.</p>
+                        </div>
+                    </div> 
+
+                    <div className="displaytext">
+                        <div className="text">
+                            <h2>Programming Languages</h2>
+                            <p>Programming languages are used to write code.<br/> Each programming language has its syntax (rules and structure) and semantics (meaning and behavior). <br/>Popular programming languages include Python, Java, C++, JavaScript, and Ruby.</p>
+                        </div>
+                        <div>
+                            <img className="programming" src="https://miro.medium.com/v2/resize:fit:820/1*v5NRqLcyXIVNaG5KLjKIQA.png" alt="" />
```

Comment:
Please add alt text for img tag

Reply:
I added text for alternate could you please review it.